docs/pages/index.vue

```diff
@@ -12,6 +12,7 @@ definePageMeta({
     <ContentFormat />
     <ContentParse />
     <ContentModify />
+    <ContentCommonHelpers />
```

Comment:
Small tweak, lets move this under <ContentData>

Reply:
Done